### PR TITLE
fix(html): sensible cloud-platform notebook html rendering

### PR DESCRIPTION
**Pull request by rivanfebrian123**
# Variables
In current version, if we click a variable, it will cause strange errors.
图片

**On Google Colab:**
Notebook: https://colab.research.google.com/drive/1cksGQIhby5ht8hOcC4szCoTSz1IgF1O3?usp=sharing
图片

**On Binder:**
Notebook: https://mybinder.org/v2/gist/rivanfebrian123/21fed85603a8ada2435d452a27de287e/HEAD?labpath=EDA%20Pandas%20Profiling.ipynb
图片

# Links
On the other hand, when we click a link, it will open inside the iframe instead of a new tab:
图片
图片

**On Google Colab:**
图片

# Solution
To fix those HTML rendering troubles, I changed the default anchoring behaviors to the more sensible ones.

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -5,18 +5,6 @@ on:
   pull_request:
 
 jobs:
```
Comment (fabclmnt) [inline .github/workflows/pull-request.yml:7]: @rivanfebrian123 Can you please revert this change?

**Review by aquemy — CHANGES_REQUESTED** [review]
Just a small rebase from develop

File: src/pandas_profiling/utils/cache.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import zipfile
```
[the remark this answers is not in the document]
Comment (rivanfebrian123) [reply src/pandas_profiling/utils/cache.py:2]: Done!

**Comment by rivanfebrian123** [comment]
*11月30日 下午 5:56 WIB*


Re-sync to `ydataai:develop` like in #1185

**Review by aquemy — APPROVED** [review]

**Pull request merged**